packages/jest-config/package.json

```
@@ -42,7 +42,8 @@
     "jest-util": "^27.3.1",
     "jest-validate": "^27.3.1",
     "micromatch": "^4.0.4",
-    "pretty-format": "^27.3.1"
+    "pretty-format": "^27.3.1",
+    "slash": "^4.0.0"
```

```suggestion
    "slash": "^3.0.0"
```

slash@4 is ESM, and we support node 10